sdk/spring/spring-messaging-azure/src/main/java/com/azure/spring/messaging/config/AzureListenerEndpointRegistrar.java

```diff
@@ -53,9 +53,9 @@ public void afterPropertiesSet() {
     }
 
     protected void registerAllEndpoints() {
-        Assert.state(this.endpointRegistry != null, "No AzureListenerEndpointRegistry set");
         synchronized (this.mutex) {
             for (AzureListenerEndpointDescriptor descriptor : this.endpointDescriptors) {
+                Assert.state(this.endpointRegistry != null, "No AzureListenerEndpointRegistry set");
```

Comment:
Why are we doing this in the for loop

Reply:
It's used for ignoring revapi checking, reverting to keep the match?